ixa-fips/src/states.rs

```diff
@@ -0,0 +1,276 @@
+/*!
+
+An enum for states as represented by FIPS Geographic Region Codes. Note that the `FIPSCode` encoded type only uses six
```

Comment:
I think it would be good to link to the spec here and what version you're using – maybe the 2020 version here https://www.census.gov/library/reference/code-lists/ansi.html?